etc/kayobe/kolla.yml

```diff
@@ -344,10 +344,6 @@ kolla_build_blocks:
     ADD additions-archive /
   magnum_base_footer: |
     RUN curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | head -n -1 | bash
```

Comment:
This is not required anymore - see https://review.opendev.org/c/openstack/kolla/+/937838